src/ecdsa/test_numbertheory.py

```diff
@@ -116,6 +173,39 @@ def st_num_square_prime(draw):
 
 
 class TestNumbertheory(unittest.TestCase):
+    def test_gcd(self):
+        assert gcd(3 * 5 * 7, 3 * 5 * 11, 3 * 5 * 13) == 3 * 5
+        assert gcd([3 * 5 * 7, 3 * 5 * 11, 3 * 5 * 13]) == 3 * 5
+        assert gcd(3) == 3
+
+    @unittest.skipUnless(HC_PRESENT,
+                         "Hypothesis 2.0.0 can't be made tolerant of hard to "
+                         "meet requirements (like `is_prime()`)")
```

Comment:
Not really sure I understand what this means, might be clarified, for people not too familiar with hypothesis

Reply:
That is, what is the actual issue ?

Reply:
when you select random integers, few of them will be primes, so the test case generation takes a lot of time; hypothesis doesn't like it when it takes few seconds

Reply:
Pleas mention that the issue is a time issue then.